src/sidebar/store/modules/test/annotations-test.js

```diff
@@ -12,14 +17,181 @@ const { actions, selectors } = annotations;
  * Create a Redux store which handles annotation, selection and draft actions.
  */
 function createStore() {
-  return createStoreFromModules([annotations, selection, drafts, viewer], [{}]);
+  return createStoreFromModules(
+    [annotations, selection, drafts, groups, session, viewer],
+    [{}]
+  );
 }
 
 // Tests for most of the functionality in reducers/annotations.js are currently
 // in the tests for the whole Redux store
 
 describe('sidebar/store/modules/annotations', function() {
-  describe('isWaitingToAnchorAnnotations', () => {
+  describe('#addAnnotations()', function() {
+    const ANCHOR_TIME_LIMIT = 1000;
+    let clock;
+    let store;
+
+    function tagForID(id) {
+      const storeAnn = store.findAnnotationByID(id);
+      if (!storeAnn) {
+        throw new Error(`No annotation with ID ${id}`);
+      }
+      return storeAnn.$tag;
+    }
+
+    beforeEach(function() {
+      clock = sinon.useFakeTimers();
+      store = createStore();
+    });
+
+    afterEach(function() {
+      clock.restore();
+    });
+
+    it('adds annotations not in the store', function() {
```

Comment:
Are we converting these to arrow functions now?

Reply:
Slowly :)